gulpfile.js

```diff
@@ -215,10 +204,46 @@ const test = gulp.series(testUnitTests, testWeb);
 const testInt = gulp.series(testDebugger);
 
 async function packageVsix() {
+    const devAiKey = '441859ca-4501-4653-bc1d-65fb8663108a'; // capisvtoolsappinsights-dev
+    const prodAiKey = '0d422197-d351-41c5-b371-a270ca3b13af'; // IsvTooling_AI_WUS2
+
     fs.emptyDirSync(packagedir);
-    return vsce.createVSIX({
+
+    // Set Preview extension name
+    await npm(['pkg', 'set', 'name=powerplatform-vscode-preview']);
+    await npm(['pkg', 'set', 'displayName="Power Platform Tools [PREVIEW]"']);
+    await npm(['pkg', 'set', 'description="Unsupported extension for testing Power Platform Tools"']);
+    await npm(['pkg', 'set', `aiKey=${devAiKey}`]);
+
+    gulp.src('README.md')
```

Comment:
all this file copying and renaming is going to be problematic.
What happens when errors occur during these tasks?

And how does the local developer build things? Will it dirty up their local git changes? since the readme.md now has a different content?

Reply:
Original readme is reset after the preview package is done, and the .gitignore update prevents "README.original.md" from polluting the git diffs

Reply:
Off of David's idea, swapped to single readme file with text replacement